packages/plugins/rubygem-foreman_remote_execution/foreman_remote_execution-10.0.6.gem

```diff
@@ -0,0 +1 @@
+../../../.git/annex/objects/F6/wJ/SHA256E-s538112--e08198b8fa5de4cb61d5a47da4738009258b3cfafd0374c2f9eb319d64c43612.6.gem/SHA256E-s538112--e08198b8fa5de4cb61d5a47da4738009258b3cfafd0374c2f9eb319d64c43612.6.gem
```

Comment:
you didn't delete the 10.0.5 gem from git, this confuses annex.

Reply:
Yeah, sorry about that, fixed